server/gc/src/main/java/org/apache/accumulo/gc/metrics/GcMetrics.java

```diff
@@ -35,7 +44,7 @@
 public class GcMetrics extends Metrics {
 
   // use common prefix, different that just gc, to prevent confusion with jvm gc metrics.
-  public static final String GC_METRIC_PREFIX = "AccGc";
```

Comment:
I think we want to be careful with modifying the metric names being emitted unless there is good reason. A change like this would cause existing consumers of the metrics to have to change all of their monitoring and alert definitions.

Reply:
Good point. I changed it to match micrometers' dot separated naming convention.

Reply:
If we have to change it to make it work, that is one thing. If we don't have to change it, then we probably shouldn't.

Reply:
That makes sense. I'll have it changed back in the next commit.